src/SqlVirtualMachine/SqlVirtualMachine.Autorest/custom/Update-AzSqlVM.ps1

```diff
@@ -318,7 +318,26 @@ param(
     [System.Management.Automation.SwitchParameter]
     # Enable automatic upgrade of Sql IaaS extension Agent.
     ${EnableAutomaticUpgrade},
-
+	
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.SqlVirtualMachine.Category('Body')]
+    [System.Management.Automation.SwitchParameter]
+    # Enable Azure Ad Authentication on SQL virtual machine.
+    ${AzureAdAuthenticationSettingEnable},
+	
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.SqlVirtualMachine.Category('Body')]
+    [System.Management.Automation.SwitchParameter]
+    # Skip client validation while enabling Azure Ad Authentication on SQL virtual machine.
+    ${AzureAdAuthenticationSkipClientValidation},	
+	
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.SqlVirtualMachine.Category('Body')]
+    [System.String]
+    # The client Id of the Managed Identity to query Microsoft Graph API.
+    # An empty string must be used for the system assigned Managed Identity
+    ${AzureAdAuthenticationSettingClientId},
```

Comment:
we have an internal discussion about the naming of the parameter.
This parameter name is subject to change.

Reply:
@BethanyZhou - I am making more changes to the naming of cmdlet and parameters. 
JFYI. I will probably close this PR and create another one.
In case the changes are taking time, we will target the next milestone.
Thanks for reviewing.